### PR TITLE
Static dashboard

### PR DESCRIPTION
**Pull request by abhishekRdl**
i have Merged changes and ran eslint for the pages that i have worked.

### CODE REVIEW

File: src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js [hunk 1]
```diff
@@ -77,6 +79,7 @@ function ConfigSetupModal({
     }, 5000);
   };
 
+  /* eslint-disable-next-line */
```
Comment (teamlead-rdl) [inline src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js:82]: Eslint disabling do it in one line at the header if the lint is of similar way

File: src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js [hunk 2]
```diff
@@ -157,7 +162,7 @@ function ConfigSetupModal({
               required
               // onBlur={() =>validateForNullValue(accessPointPassword, 'accessPointPassword')}
               onChange={(e) => { setAccessPointPassword(e.target.value); }}
-              autoComplete="off"
+              autoComplete="new-password"
```
Comment (teamlead-rdl) [inline src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js:165]: Why is it marked as New-password? Is it a requirement?

File: src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js [hunk 3]
```diff
@@ -196,10 +199,9 @@ function ConfigSetupModal({
               label="password"
               variant="outlined"
               fullWidth
-              
               //  onBlur={() =>validateForNullValue(ftpPassword, 'ftpPassword')}
               onChange={(e) => { setFtpPassword(e.target.value); }}
-              autoComplete="off"
+              autoComplete="new-password"
```
Comment (teamlead-rdl) [inline src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js:204]: "New-password" is it in requirement?

File: src/components/Device/subComponent/BumpTestComponentModal.js [hunk 4]
```diff
@@ -144,17 +145,25 @@ function BumpTestComponentModal({
           }
           if (dataCount < 3) {
             setPercentageDeviation('NA');
+            setLoading(false);
+            setProgress('start');
           } else {
             let avg = 0;
             avg = tot / dataList;
```
Comment (teamlead-rdl) [inline src/components/Device/subComponent/BumpTestComponentModal.js:152]: Please correct the spell of tot to something meaningful

**Review by teamlead-rdl — CHANGES_REQUESTED** [review]

File: src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js (hunk 2 above)
Comment (abhishekRdl) [reply src/components/Device/deviceConfiguration/subcomponent/ConfigSetupModalComponent.js:165]: Hi sir, AutoComplete was  not working in edge, when i googled it , it came we need to add new-password

File: src/components/Device/subComponent/BumpTestComponentModal.js (hunk 4 above)
Comment (teamlead-rdl) [reply src/components/Device/subComponent/BumpTestComponentModal.js:152]: @abhishekRdl , please push theis change

Comment (abhishekRdl) [reply src/components/Device/subComponent/BumpTestComponentModal.js]: I have done changes as told by you

**Pull request closed without merging**